### DISCUSSION TITLE
为什么要把visible改成open?

### DISCUSSION THREAD

**Discussion by translateServers** [ant-design/ant-design#37456] [Q&A]
### What problem does this feature solve?

要用新版本之前写的visible全都要改，而且open更像一个动作不是一个状态，一点不符合常识

### What does the proposed API look like?

visible同样支持

**Comment by wulibaibao** [comment]
open = 键盘按4下 
visible = 键盘按7下
开发少打3个字，单输出提升43%开发速度

**Reply by crazyair** [reply]
这个单词的字母，跟开发没有关系把

**Comment by zombieJ** [comment]
[FAQ: 弹层类组件为什么要统一至 open 属性？](https://ant.design/docs/react/faq-cn#%E5%BC%B9%E5%B1%82%E7%B1%BB%E7%BB%84%E4%BB%B6%E4%B8%BA%E4%BB%80%E4%B9%88%E8%A6%81%E7%BB%9F%E4%B8%80%E8%87%B3-open-%E5%B1%9E%E6%80%A7%EF%BC%9F)

**Reply by crazyair** [reply]
刚看到 Image 还有
